examples/experimental/persistent-volume-provisioning/glusterfs-dp.yaml

```diff
@@ -6,6 +6,6 @@ provisioner: kubernetes.io/glusterfs
 parameters:
   endpoint: "glusterfs-cluster"
   resturl: "http://127.0.0.1:8081"
-  restauthenabled: "true"
-  restuser: "admin"
-  restuserkey: "password"
\ No newline at end of file
+  userId: "admin"
+  secretNamespace: "default"
+  secretName: "heketi-secret"
```

Comment:
May be this can be renamed to 'gluster-secret' or 'rest-secret' for name consistency among other fields as I chose  generic name for all the other fields.